app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+class Animal:
+    alive: list = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name: str = name
+        self.health: int = health
+        self.hidden: bool = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, Hidden: {self.hidden}}}")
+
+    @classmethod
+    def remove_dead(cls) -> None:
+        cls.alive = [animal for animal in cls.alive if animal.health > 0]
+
+    @classmethod
+    def reset_alive(cls) -> None:
+        cls.alive.clear()
+
+    @classmethod
+    def __str__(cls) -> str:
+        return str(cls.alive)
```

Comment:
The `__str__` method should be an instance method rather than a class method, so it can represent the string representation of a single instance, not the class itself. Remove the @classmethod decorator.